test/distributed/tensor/test_utils.py

```diff
@@ -766,6 +774,59 @@ def test_2d_mesh_uneven_strided_shard(self):
             self.assertEqual(dtensor.full_tensor(), tensor)
 
 
+class Test_StridedShard_with_shard_order(LocalDTensorTestBase):
+    @property
+    def world_size(self) -> int:
+        return 32
+
+    @with_comms
```

Comment:
...if this is under LocalTensorMode why do you need `with_comms`? (cc @dzmitry-huba )